web/css/gde.css

```
@@ -1,1562 +1,2373 @@
+	/* Forma */
+	--r-sm: 4px;
+	--r:    7px;
+	--r-lg: 11px;
+	--ring: 0 0 0 3px rgba(22, 125, 155, .18);
+
+	--shadow-sm: 0 1px 2px rgba(14, 40, 51, .06);
+	--shadow:    0 1px 3px rgba(14, 40, 51, .07), 0 4px 14px rgba(14, 40, 51, .05);
+	--shadow-lg: 0 8px 30px rgba(14, 40, 51, .14);
+
+	/* Tipografia */
+	--font: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)